document-generator-core/swagger-2.0/models/document-generator.json

```diff
@@ -0,0 +1,76 @@
+{
+  "definitions": {
+    "documentGeneratorRequest": {
```

Comment:
name should be `documentRequest` to describe the data we're sending in, the title below should then be `DocumentRequest`